### PR TITLE
Chore: Deprecate FolderIds from FindPersistedDashboardsQuery

### PR DESCRIPTION
**Pull request by yangkb09**
Deprecate `FolderIds` in `FindPersistedDashboardsQuery` struct. Part of a series of PRs to deprecate sequential folder IDs.

Ref https://github.com/grafana/grafana/issues/61232

### CODE REVIEW

File: pkg/services/dashboards/database/database_folder_test.go [hunk 1]
```diff
@@ -170,7 +170,7 @@ func TestIntegrationDashboardFolderDataAccess(t *testing.T) {
 						FolderIds: []int64{
 							rootFolderId,
 							folder1.ID,
-						},
+						}, // nolint:staticcheck
```
Comment (suntala) [inline pkg/services/dashboards/database/database_folder_test.go:173]: Will this work? I wonder if this should be attached to the line `FolderIds: []int64{`.

File: pkg/services/dashboards/models.go [hunk 2]
```diff
@@ -483,13 +483,14 @@ type FindPersistedDashboardsQuery struct {
...
 	DashboardUIDs []string
 	Type          string
-	FolderIds     []int64
-	FolderUIDs    []string
-	Tags          []string
-	Limit         int64
-	Page          int64
-	Permission    PermissionType
-	Sort          model.SortOption
+	// Deprecated: use FolderUID instead
```
Comment (suntala) [inline pkg/services/dashboards/models.go:486]: Nit: `Deprecated: use FolderUIDs instead`

**Review by suntala — COMMENTED** [review]
Just had a couple of small notes/questions but looks ready to go after that 👍

File: pkg/services/dashboards/models.go (hunk 2 above)
Comment (suntala) [inline pkg/services/dashboards/models.go:486]: Nit:
```suggestion
	// Deprecated: use FolderUIDs instead
```

File: pkg/services/dashboards/database/database_folder_test.go (hunk 1 above)
Comment (yangkb09) [reply pkg/services/dashboards/database/database_folder_test.go:173]: it seems like it works since drone doesn't complain 🤔 , but good to note

**Review by suntala — APPROVED** [review]

**Pull request merged**